src/simplewallet/simplewallet.cpp

```diff
@@ -2827,6 +2876,13 @@ bool simple_wallet::new_wallet(const boost::program_options::variables_map& vm,
     return false;
   }
 
+  if (!m_subaddress_lookahead.empty())
+  {
+    auto lookahead = parse_subaddress_lookahead(m_subaddress_lookahead);
+    assert(lookahead);
```

Comment:
Can this be an error rather than what is essentially a crash exit ?

Reply:
The parse failure is checked earlier, so this parse here is guaranteed to succeed. Do you still want an (impossible) error handling code here?